src/main/java/io/vertx/pgclient/codec/encoder/MessageEncoder.java

```diff
@@ -39,6 +39,10 @@ protected void encode(ChannelHandlerContext ctx, Message message, ByteBuf out) t
       writeCString(out, "vertx-pg-client", UTF_8);
       writeCString(out, "client_encoding", UTF_8);
       writeCString(out, "utf8", UTF_8);
+      writeCString(out, "DateStyle", UTF_8);
+      writeCString(out, "ISO", UTF_8);
+      writeCString(out, "extra_float_digits", UTF_8);
+      writeCString(out, "2", UTF_8);
```

Comment:
> **Java issues**
> If you happen to use Java as the frontend, there are some points that have to be
> taken into consideration. Java tends to pass some parameters to the server as part
> of the connection string. One of those parameters is `extra_float_digits`. This
> `postgresql.conf` parameter governs the floating-point behavior of PostgreSQL,
> and is set by Java to make things more deterministic.

[PostgreSQL Replication - Second Edition](https://www.packtpub.com/big-data-and-business-intelligence/postgresql-replication-second-edition)